FirstClasses/multiples/src/multiples.erl

```diff
@@ -1,5 +1,39 @@
 -module(multiples).
 
 -export([thousand_sum/0]).
+-export([generate_input_collection/1]).
+-export([filter_collection/1]).
 
-thousand_sum() -> 1.
+thousand_sum() -> 
+    FilterCollection = filter_collection(generate_input_collection(1000)),
+    lists:sum(FilterCollection).
+
+generate_input_collection(Lenght) ->
+    case Lenght of
+        0  ->
+           [];
+        L when L - 1 == 0 ->
+            1;
```

Comment:
I guess you meant  `[1]` instead of `1`, as the returned value is supposed to be a (one element) list and not an integer ;)